data/json/itemgroups/Weapons_Mods_Ammo/guns_by_calibre.json

```diff
@@ -84,6 +84,15 @@
     "id": "guns_pistol_common_10mm",
     "items": [ { "group": "nested_sw_610", "prob": 5 } ]
   },
+  {
+    "type": "item_group",
+    "id": "guns_pistol_common_762_39",
+    "items": [
+      { "group": "nested_draco", "prob": 10 },
```

Comment:
Can you show us a screenshot of the search results?

Reply:
![image](https://github.com/user-attachments/assets/c0646558-56c9-4cb7-9ebc-add41ec59e2f)

Reply:
You should be able to look at barrel lengths, too?

Reply:
Good to resolve this?